docs/content/gameobject/readers-writers.md

```diff
@@ -26,7 +26,10 @@ Readers and Writers allow you to inspect and change the state of SpatialOS compo
 
   * Change the property values of a SpatialOS component.
   * Send events defined in a SpatialOS component.
-  * All the functionality that a Reader provides (except for listening to authority state changes)
+  * Send acknowledgement for [`AuthorityLossImminent` notifications](https://docs.improbable.io/reference/latest/shared/design/understanding-access#enabling-and-configuring-authoritylossimminent-notifications).
```

Comment:
This either needs an indefinite article or for acknowledgement to be pluralised.

Reply:
Agreed